templates/articles/to_translate.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block body %}
+<section>
+  <h1>Articles to translate in {{ current_language[1] }}</h1>
+
+  <p>Change language:</p>
+  <ul>
+    {% for language in get_languages() %}
+      {% if current_language[0] != language[0] %}
+        <li><a href={{ url_for('articles.to_translate') }}?in={{language[0]}}>{{ language[1] }}</a></li>
+      {% endif %}
+    {% endfor %}
+  </ul>
+
+  <ul>
+  {% for article in articles %}
+    <li>
+      <article>
+        <h2>{{ article.title }}</h2>
+        <ul>
```

Comment:
I suppose a `<nav>` would be more appropriate but we see with @cafebabel-webmaster when implementing the frontend.